gui/js/onboarding.window.js

```diff
@@ -36,6 +36,24 @@ module.exports = class OnboardingWM extends WindowManager {
     }
   }
 
+  jumpToSyncPath () {
+    this.shouldJumpToSyncPath = true
+    this.send('registration-done')
```

Comment:
Remove the first `registration-done` event? Or add a `TODO` notice? Do we have a spinner on the `Use Cozy Drive` onboarding button?

Reply:
TODO added